pages/src/common/badge.less

```diff
@@ -34,6 +34,11 @@
     background-color: #dc3545;
   }
 
+  &-success {
+    color: #212529;
+    background-color: #04c250;
+  }
```

Comment:
Maybe it makes sense to create variables with all these colors in pages\src\common\colors.less